web/src/classic/components/organisms/EarthEditor/Header/index.tsx

```diff
@@ -51,8 +51,8 @@ const Header: React.FC<Props> = ({ className }) => {
         currentProjectStatus={projectStatus}
         currentProject={currentProject}
         user={user}
-        workspaces={teams}
-        teamId={teamId}
+        workspaces={workspaces}
+        teamId={workspaceId}
```

Comment:
why are we passing `workspaceId` with the `teamId`? Please fix the name so that its consistent.

Reply:
done